addons/sale_timesheet/report/project_profitability_report_analysis.py

```diff
@@ -181,18 +181,18 @@ def init(self):
                                     LEFT JOIN sale_order_line SOL ON SOINV.order_line_id = SOL.id
                                     LEFT JOIN account_move_line AML ON AAL.move_id = AML.id
                                                                    AND AML.parent_state = 'posted'
-                                                                   AND AML.exclude_from_invoice_tab = 'f'
+                                                                   AND COALESCE(AML.line_type LIKE 'invl%%', 'f')
```

Comment:
`'f'` is a literal string. Whatever you are trying to do, it's wrong. A `coalesce` here is a non-sense.

```suggestion
                                                                   AND AML.line_type LIKE 'invl%%'
```

Same remark for all changes in this query.